# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/rust/naga/src/back/spv/mod.rs
# Commit: b3109ad14685
# Full Hash: b3109ad14685624ada7861083ff5332aec9a7818
# Author: Andy Leiserson <aleiserson@mozilla.com>
# Date: 2025-10-07 17:00 +0000
# Description:
#   Bug 1992842 - Update wgpu to upstream 1072b878 (2025-10-06) r=webgpu-reviewers,supply-chain-reviewers,ErichDonGubler
# ==============================================================================

diff -r 1d3f000cf1f9 -r b3109ad14685 third_party/rust/naga/src/back/spv/mod.rs
--- a/third_party/rust/naga/src/back/spv/mod.rs	Tue Oct 07 16:51:44 2025 +0000
+++ b/third_party/rust/naga/src/back/spv/mod.rs	Tue Oct 07 17:00:40 2025 +0000
@@ -26,7 +26,6 @@
 use thiserror::Error;
 
 use crate::arena::{Handle, HandleVec};
-use crate::path_like::PathLikeRef;
 use crate::proc::{BoundsCheckPolicies, TypeResolution};
 
 #[derive(Clone)]
@@ -96,7 +95,7 @@
 #[derive(Debug, Clone)]
 pub struct DebugInfo<'a> {
     pub source_code: &'a str,
-    pub file_name: PathLikeRef<'a>,
+    pub file_name: &'a str,
     pub language: SourceLanguage,
 }
 